module/VuFind/src/VuFind/Session/Database.php

```diff
@@ -87,6 +87,7 @@ public function destroy($sessId)
      *
      * @return bool
      */
+    #[\ReturnTypeWillChange]
```

Comment:
I see from the spec that as of PHP 7.1, this should be returning the count of rows deleted rather than true. Should we try to address that in a separate PR, or at least open a JIRA ticket about it?

Reply:
A JIRA ticket is probably the only option, because the type in PHP 8.1 is a union `int|false` that's not supported pre-PHP 8.0.0.

Reply:
But you're right that the actual return value could be corrected right away. I'd do that in a separate PR.

Reply:
JIRA issue for the return type hint: https://openlibraryfoundation.atlassian.net/browse/VUFIND-1557

Reply:
Thanks!